csi.proto

```diff
@@ -146,6 +146,11 @@ message CreateVolumeRequest {
   // This field is OPTIONAL. The Plugin is responsible for parsing and
   // validating these parameters. COs will treat these as opaque.
   map<string, string> parameters = 5;
+
+  // End user credentials used to authenticate/authorize volume creation
+  // request.
+  // This field is OPTIONAL.
+  Credentials userCredentials = 6;
```

Comment:
Let's follow protobuf style guide and use snake_case for field names:
https://developers.google.com/protocol-buffers/docs/style